packages/core/src/components/Toast/Toast.tsx

```diff
@@ -102,7 +105,7 @@ const Toast: FC<ToastProps> & { types?: typeof ToastType; actionTypes?: typeof T
 
       clearTimeout(timerAutoHide.current);
       timerAutoHide.current = setTimeout(() => {
-        handleClose();
+        handleClose(ToastCloseReason.TIMEOUT);
```

Comment:
let's convert `ToastCloseReason` to a type, and instead of accessing it through `ToastCloseReason`, just send "timeout" there.
so `ToastCloseReason` should look like:
```
type ToastCloseReason = "timeout" | "close-click"
```